tests/base.py

```diff
@@ -4,8 +4,7 @@
 from datetime import datetime as dt
```

Comment:
test_gsc_start_date.py file has one print line L54 can you replace the same with LOGGER.info?